sandbox_emulator/.env copy

```diff
@@ -0,0 +1,12 @@
+DEVICE_IP=18.102.29.37
```

Comment:
this file should not be commited, it's name should be .env.sample.{device_name} to be ignored